projects/observability/src/pages/apis/service-detail/service-detail-breadcrumb.resolver.ts

```diff
@@ -54,6 +54,10 @@ export class ServiceDetailBreadcrumbResolver implements Resolve<Observable<Bread
   }
 
   private getAttributeKeys(): string[] {
-    return ['name'];
+    return ['name', ...this.getExtraAttributeKeys()];
+  }
+
+  protected getExtraAttributeKeys(): string[] {
```

Comment:
nit: I think it would make more sense to make `getAttributeKeys` protected rather than expose this unused function

Reply:
Thats what I had done in the first iteration. Then, @skjindal93 made a point that we can avoid repeating the base list/keys (just 'name' in this case) if we take this route.

Any strong reasons to prefer either way? I couldn't think of any except for probably the flexibility to override the whole list.